AGENTS.md

```
@@ -0,0 +1,28 @@
+# Agent Instructions
+
+Instructions for AI coding agents (Claude Code, CodeRabbit, etc.) working in this repository.
+
+## Commit messages
+
+Use [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)